MainRepository/Program.cs

```diff
@@ -1,7 +1,215 @@
+               if (key.Key is ConsoleKey.Backspace or ConsoleKey.LeftArrow) {
+                  if (j == 0) j--;
+                  else {
+                     mWordle[i, j - 1] = mCircle;
+                     mWordle[i, j] = mDot;
+                     j -= 2;
+                     mList.RemoveAt (mList.Count - 1);
+                  }
+               } else if (char.ToUpper (key.KeyChar) is >= 'A' and <= 'Z') {
+                  if (j < 4) mWordle[i, j + 1] = mCircle;
+                  char c = Convert.ToChar (key.KeyChar);
+                  mWordle[i, j] = char.ToUpper (c);
+                  if (mList.Count < 5) mList.Add (c);
+               }
+               if (j == 4) {
+                  DisplayBoard (word);
+                  while (true) {
+                     var lastKey = Console.ReadKey (true);
+                     if (lastKey.Key == ConsoleKey.Enter && ValidWord ()) {
+                        Console.WriteLine (new string (' ', Console.WindowWidth));
+                        break;
+                     }
+                     if (lastKey.Key == ConsoleKey.Enter && !ValidWord ()) {
+                        mBadWord = string.Join ("", mList).ToUpper ();
+                        var cursorTop = mIsInvalid ? Console.CursorTop - 1 : Console.CursorTop;
+                        Console.SetCursorPosition (mHalfWidth - 1, cursorTop);
+                        Console.ForegroundColor = ConsoleColor.Yellow;
+                        Console.WriteLine ($"{mBadWord} is not a word");
+                        Console.ResetColor ();
+                        mIsInvalid = true;
+                     }
+                     if (lastKey.Key == ConsoleKey.Backspace) {
+                        mWordle[i, 4] = mCircle;
+                        mWordle[i + 1, 0] = mDot;
+                        j--;
+                        mList.RemoveAt (4);
+                        Console.SetCursorPosition (mHalfWidth - 1, Console.CursorTop - 1);
+                        mIsInvalid = false;
+                        Console.WriteLine (new string (' ', Console.WindowWidth));
+                        break;
+                     }
+                  }
+               }
+               if (i < 5 && j == 4) mWordle[i + 1, 0] = mCircle;
+               DisplayBoard (word);
+               if (mList.Count == 5) {
+                  DisplayBoard (word, true);
+                  if (mGreenList.Count == 5) {
+                     mGameOver = true;
+                     mCount++;
+                     return;
+                  }
+                  if (!ValidWord ()) j--;
+                  else mList.Clear ();
+               }
+            }
+         }
+         mCount++;
+      }
+   }
+
+   /// <summary>Sets the foreground color for the alphabet</summary>
+   public char SetForegroundClr (char ch) {
+      if (mGreenList.Contains (ch)) Console.ForegroundColor = ConsoleColor.Green;
+      else if (mBlueList.Contains (ch))

```

Comment:
Make these as single lines
```suggestion
      else if (mBlueList.Contains (ch)) Console.ForegroundColor = ConsoleColor.Blue;

```